pkg/ffmpeg/stream_transcode.go

```diff
@@ -64,8 +104,8 @@ var (
 	}
 	StreamTypeMKV = StreamFormat{
 		MimeType: MimeMkvVideo,
-		Args: func(videoFilter VideoFilter, videoOnly bool) (args Args) {
-			args = args.VideoCodec(VideoCodecCopy)
+		Args: func(codec VideoCodec, videoFilter VideoFilter, videoOnly bool) (args Args) {
+			args = CodecInit(codec)
```

Comment:
For MKV the codec should stay as copy - that just copies the stream directly and does no transcoding at all, so using hardware acceleration would actually negatively impact performance.